src/core/algorithms/cfd/cfdfinder/model/hyfd/validator.cpp

```diff
@@ -0,0 +1,323 @@
+#include "validator.h"
+
+#include <algorithm>
+#include <future>
+#include <utility>
+#include <vector>
+
+#include <boost/asio/post.hpp>
+#include <boost/asio/thread_pool.hpp>
+#include <easylogging++.h>
+
+#include "algorithms/fd/hycommon/util/pli_util.h"
+#include "algorithms/fd/hycommon/validator_helpers.h"
+#include "algorithms/fd/hyfd/hyfd_config.h"
+#include "types/bitset.h"
+
+namespace {
+using BitSet = boost::dynamic_bitset<>;
```

Comment:
You've included `model/types/bitset.h` (btw, I can't see its usages in this file), which defines `Bitset` class, and defined `BitSet` here.
It's very likely that these types will be misused. I suggest renaming this to

```suggestion
using DynBitSet = boost::dynamic_bitset<>;
```